packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -149,6 +149,10 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Thu Jun 1 2017 Adnan Khan <adnkha@amazon.com> - 1.14.2-1
+- Cache Agent version 1.14.2
+- Add functionality for running agent with userns=host when Docker has userns-remap enabled
+- Add support for Docker 17.03.1ce
```

Comment:
turns out we *have* to call it "ce" in the spec file, otherwise it makes rpm versioning sad.